tests/test.py

```diff
@@ -0,0 +1,4 @@
+bla1
```

Comment:
(mypy): Name "bla1" is not defined  [name-defined]  <!-- Automatically posted by Lintly -->